packages/phone-number-privacy/signer/src/index.ts

```diff
@@ -1,16 +1,9 @@
 import * as functions from 'firebase-functions'
 import logger from './common/logger'
-import { handleGetContactMatches } from './match-making/get-contact-matches'
 import { handleGetBlindedMessageForSalt } from './salt-generation/get-salt'
 
 // EG. curl -v "http://localhost:5000/celo-phone-number-privacy/us-central1/getBlindedSalt" -H "Authorization: 0xdaf63ea42a092e69b2001db3826bc81dc859bffa4d51ce8943fddc8ccfcf6b2b1f55d64e4612e7c028791528796f5a62c1d2865b184b664589696a08c83fc62a00" -d '{"hashedPhoneNumber":"0x5f6e88c3f724b3a09d3194c0514426494955eff7127c29654e48a361a19b4b96","blindedQueryPhoneNumber":"n/I9srniwEHm5o6t3y0tTUB5fn7xjxRrLP1F/i8ORCdqV++WWiaAzUo3GA2UNHiB","account":"0x588e4b68193001e4d10928660aB4165b813717C0"}' -H 'Content-Type: application/json'
 export const getBlindedSalt = functions.https.onRequest(async (request, response) => {
   logger.info('Begin getBlindedSalt request')
   return handleGetBlindedMessageForSalt(request, response)
 })
-
-// EG. curl -v "http://localhost:5000/celo-phone-number-privacy/us-central1/getContactMatches" -H "Authorization: <SIGNED_BODY>" -d '{"userPhoneNumber": "+99999999999", "contactPhoneNumbers": ["+5555555555", "+3333333333"], "account": "0x117ea45d497ab022b85494ba3ab6f52969bf6812"}' -H 'Content-Type: application/json'
-export const getContactMatches = functions.https.onRequest(async (request, response) => {
```

Comment:
one thing to note is that by keeping the contact matching in the combiner, the combiner still needs to maintain a db `accounts` table.  it will only use/update the `didMatchmaking` field for a record, but will have inaccurate `numlookups` info.  

this currently won't cause any issues in the performance, but it could cause confusion down the line, since it would be out of sync (not even maintaining) this field.  

In the ideal case, I wonder:
1. should the combiner have its own db table for matchmaking (without this access field)?
or
2. should the getBlindedSalt services return the number of queries they calculated for the request and then the combiner updates its table to include this field?


Also for future use of the matchmaking, the only way to give free queries will be if the matchmaking is done by the signers and then the results combined in the combiner.  By removing this now from the signer, will we have difficulty having partners update to perform this request as part of the service later?

Reply:
for documentation purposes, the decision made moving forward will be to update the table for combiner to not include query counts.  the combiner may be a distributed service in the future and keeping these in sync will become very difficult.  because of this and for "free matchmaking queries" the logic and table for matchmaking will likely also be moved to the signers, so combiner would not maintain state.